### DISCUSSION TITLE
使用 Sandboxie 实现 ClassIsland 多行显示

### DISCUSSION THREAD

**Discussion by LyCecilion** [ClassIsland/ClassIsland#370] [展示台]
> # 双岛
>
> 那是两座几乎完全对称的岛屿。
>
> 如孪生兄弟一般地——无人知道，它们是否是过去失散的浪子，又或者只是大自然万里挑一的巧合。
>
> 至少，它们现在见面了，不是吗？或许诞生时，它们即分居于遥远的地球两端；又或者亦曾是一体的陆地，却因鲜知的原因飘零又聚集。无论如何，它们暂时结束了这无尽岁月中的漂泊，在蔚蓝的大海上相遇——不，或许是「重逢」？
>
> 那毕竟只是孤岛。人类眼中的孤岛。植物繁茂生长、却人迹罕至的孤岛。探索这一切是否具有意义？毕竟，现在的世人并不总是将目光投向这两座岛屿。毕竟，那只是两座孤岛罢了——那些曾经被世人认为不可得兼的二者——或许，一曰晨曦，一曰暮霭？为炽热的日出——唤醒生机活力的群峦；为橘红的日落——笼罩那即将安眠的矿野。
>
> 毕竟，那只是孤岛。无人岛。或许有探险家曾涉足这里，或惊异于这并不符合审美的风景；或只是留下自己的一步足迹，便匆匆离去而前往自己的目的地。总之，两座孤岛就是鲜有人知。它们未曾是人类着眼于之处。这两座岛并不构成人类探索的全部。
>
> 但它们自身便是它们的全部。
>
> 毕竟，在无人问询的地方，一切都照旧运行着呢——岛上自有动植物能窥见——
>
> 那金灿灿的阳光落在晨曦岛之上——群鸟盘旋在植物上，哼出婉转动听的歌儿，悠扬而轻快地讲述着孤独而崇高的故事。那橘红色的日落，颜料般泼洒在绿油油的旷野上——然后在紧邻的夜晚，星移斗转，群星共辉。整个宇宙似乎皆归于此。
>
> 它们又需要什么所谓人类探索的意义？它们的存在便是意义。
>
> 与「孤芳自赏」不似地，它只是存在。静静伫立。它或许能听闻世俗的耳语，但它就是孤岛，不可改变的孤岛。孤岛成就了它的独特，而非它趋于独特的心理造就了它孤岛的身份。
>
> 或许再过几千几万年，这里会出现海峡和定居的人们。晨曦岛上是金色的沙滩，孩子们在沙滩上自由追逐嬉戏；暮霭岛上是轻快的海岸，人们点起篝火，围坐着，有说有笑。
>
> 徜徉在幸福中的人类，或许已经不会在思考所谓「意义」了。
>
> 毕竟，他们的存在，也是他们自己的意义。
>
> 是他人和他物无法动摇的，独属于自己的意义。
>
> 海面亮起点点灯火，那是人类的船只穿梭在这其中。他们载着希望和梦想，穿梭过光和影，为一切晨曦送去暮霭，为一切暮霭带来晨曦。

![使用 Sandboxie 实现 ClassIsland 多行显示](https://github.com/user-attachments/assets/52b18127-9c4c-4d61-8c35-ab53d68b5484)

**Comment by PrefacedCorg** [comment]


**Reply by LyCecilion** [reply]
我在写晨曦与暮霭的交织生出的梦境。我在写两个孤独的灵魂绽放最原始的生命。

**Reply by PrefacedCorg** [reply]
> 我在写晨曦与暮霭的交织生出的梦境。我在写两个孤独的灵魂绽放最原始的生命。

666

**Comment by LyCecilion** [comment]
已知 Bug：Sandboxie 上的 ClassIsland 默认配置下，无法做到鼠标置于其顶部时隐藏。

不知道更改 RawInput 等相关设置是否有用？等待随后测试。

**Reply by PrefacedCorg** [reply]
你顺便测下能否自启（你应该会使用sandboxie吧）

**Reply by LyCecilion** [reply]
> 你顺便测下能否自启（你应该会使用sandboxie吧）

理论上没问题吧……只要 Sandboxie 能自启，两个 ClassIsland 就能自启

**Comment by jiangyin14** [comment]
恭喜恭喜，已官方实现🎉
